File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    return [...my_string].filter(el => Number(el) == el).map(Number).sort((a, b) => a - b);
}

// 정규식 사용해서 처리
// 혹은 형변환으로 처리
// 숫자만 뽑아내서 정렬 후 반환

File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.kt
class Solution {
    fun solution(my_string: String): List<Int> {
        val numbers = mutableListOf<Int>()
        for (char in my_string) {
            if (char.isDigit()) {
                numbers.add(char.toString().toInt())
            }
        }

        numbers.sort()

        return numbers
    }
}